projects/plugins/jetpack/modules/masterbar/wp-posts-list/bootstrap.php

```diff
@@ -0,0 +1,12 @@
+<?php
+/**
+ * WP-Admin Posts list bootstrap file.
+ *
+ * @package Jetpack
+ */
+
+use Automattic\Jetpack\Dashboard_Customizations\Posts_List_Page_Notification;
+
+require_once __DIR__ . '/class-posts-list-page-notification.php';
+
+Posts_List_Page_Notification::init();
```

Comment:
Can we run this only Pages Screen?

```suggestion
require_once __DIR__ . '/class-posts-list-page-notification.php';

Posts_List_Page_Notification::init();
```
```suggestion
add_action( 'current_screen', 'posts_list_page_notification_init' );

function posts_list_page_notification_init() {
    $currentScreen = get_current_screen();
    if( $currentScreen->id === "edit-page" ) {
        require_once __DIR__ . '/class-posts-list-page-notification.php';

         Posts_List_Page_Notification::init();
    }
}
```

Reply:
I think we should be left on all pages because it would prevent users to access the page directly. Besides this, [WPCore does a similar thing](https://core.trac.wordpress.org/browser/trunk/src/wp-includes/capabilities.php#L87) (but instead of removing the access altogether it just adds manage_options).

Reply:
In that case we could also add a condition for the editor. I don't think that letting this code run on every screen would benefit us.

Reply:
Yeah, I do see your point. Normally, creating another object and require-ing another script wouldn't be a major problem since PHP (especially the latest versions) has a lot of caching features. The problem might be with that get_option call. 

You know what, I'll add a simple check for the $pagenow global variable checking if the screen is 'edit.php' or 'post.php'.